USAGE.md

```diff
@@ -236,6 +236,8 @@ name: Daniel S. Himmelstein  # mandatory
 initials: DSH  # optional
 orcid: 0000-0002-3012-7446  # mandatory
 twitter: dhimmel  # optional
+mastodon: dhimmel  # optional: mastodon username
+mastodon-server: mastodon.social  # optional: mastodon server (instance)
```

Comment:
```suggestion
mastodon-server: genomic.social  # optional: mastodon server (instance)
```
I believe this was the previous suggestion to use @dhimmel's real account in the example metadata